user_docs/en/userGuide.t2t

```diff
@@ -1220,6 +1220,16 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as Microsoft Word, where formatting is important.
 
+==== Report line indents With ====
+the Report line indents with combo box has four options.

```

Comment:
I feel this introductory sentence should possibly explain the point of the choice here; e.g. "This option allows you to configure how indentation at the beginning of liens is reported."